<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCreationDialog</class>
 <widget class="QDialog" name="ProjectCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Creation</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTitle"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pathLayout">
     <item>
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
